src/main/java/address/controller/RootLayoutController.java

```diff
@@ -82,7 +75,7 @@ private FileChooser getXmlFileChooser() {
     @FXML
     private void handleNew() {
         logger.debug("Wiping current model data.");
-        PrefsManager.getInstance().clearSaveLocation();
+        modelManager.clearPrefsSaveLocation();
```

Comment:
I am not sure. Introduced in #91. I am guessing when you create a new addressbook you want to clear the save file so it won't erase the current addressbook file.

Should I remove it?